sycl/doc/extensions/MemChannel/MemChannel.asciidoc

```diff
@@ -45,7 +61,7 @@ Add a new property to Table 4.33: Properties supported by the SYCL buffer class
 [options="header"]
 |===
 | Property | Description
-| property::buffer::mem_channel | The `mem_channel` property is a hint to the SYCL runtime that the buffer should be stored in a particular memory channel provided to the property.  
+| property::buffer::ext_intel_mem_channel | The `ext_intel_mem_channel` property is a hint to the SYCL runtime that the buffer should be stored in a particular memory channel provided to the property.  
```

Comment:
@Pennycook, @GarveyJoe: Do we want to name the property like this, or should it be in a separate namespace like `sycl::ext::intel::property::mem_channel`?  I suspect we will want it in the new namespace eventually.  Is there some advantage to keeping it in the existing namespace for now and moving it later, or should we just put it in the new namespace now?

Reply:
This extension spec defines an "old-style" property.  Once we implement "new-style" properties, will we want **both** a new-style property and also an old-style property for `mem_channel`?  If so, that might be a reason not to define this property as `property::buffer::mem_channel` (because we would reserve that name for the new-style property).

Reply:
@gmlueck @Pennycook the spec for new property is not ready yet, am I right that we can keep current impl as is? (we have deprecated `property::buffer::mem_channel` and new `property::buffer::ext_intel_mem_channel`), and in the release after the upcoming release we will remove `property::buffer::mem_channel`, so the impl of current spec will have only `property::buffer::ext_intel_mem_channel`, and the impl of new spec will have `sycl::ext::intel::mem_channel`

Reply:
In order to avoid requiring our customers to change their code twice, how about this:

* Leave `property::buffer::mem_channel` undeprecated for now, even though it does not conform to SYCL 2020 naming conventions.
* Do NOT add `property::buffer::ext_intel_mem_channel`.
* Once we implement the new properties, add `sycl::ext::intel::property::mem_channel` and deprecate `property::buffer::mem_channel`.

Reply:
Oh, thanks, now it's clear.
@romanovvlad, is it ok to NOT align mem_channel property extension with SYCL 2020 in the upcoming release and deprecate it later in accordance with Greg's plan above?

Reply:
>> @romanovvlad, is it ok to NOT align mem_channel property extension with SYCL 2020 in the upcoming release and deprecate it later in accordance with Greg's plan above?

That's OK.

Reply:
Reverted in https://github.com/intel/llvm/pull/4432/commits/d849467ac1826aaee7997d97850825c5182c6c0c

Reply:
Sorry for the confusion, but I was *not* suggesting that we revert all changes to this extension.  I was only suggesting that we keep the property name as `property::buffer::mem_channel` rather than changing it to `property::buffer::ext_intel_mem_channel`.  All the other changes you made to this extension should be retained.

Reply:
OK, will do

Reply:
Returned back here: https://github.com/intel/llvm/pull/4432/commits/198aacaf7a134a0979f2147d2b5711747593e618